tools/sof-get-default-tplg.sh

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
+#
+# read kernel log to get topology file loaded by SOF driver
+# Example from an apl up2 pcm512x device:
+#
+# sof-audio-pci 0000:00:0e.0: loading topology:intel/sof-tplg/sof-apl-pcm512x.tplg
+#
+# sof-apl-pcm512x.tplg will be returned
+#
 
-# Current: load TPLG file order:
-# dmesg
-# journalctl
-# /var/log/syslog (sof-kernel-dump.sh)
-# Future: possibly be loaded from elsewhere
-# Notice: Only verified on Ubuntu 18.04
-tplg_file=$(dmesg |grep -i topology|awk -F ':' '/tplg/ {print $NF;}'|tail -n 1)
-[[ ! "$tplg_file" ]] && \
-    tplg_file=$(journalctl -k |grep -i topology |awk -F ':' '/tplg/ {print $NF;}'|tail -n 1)
-[[ ! "$tplg_file" ]] && \
-    tplg_file=$(sof-kernel-dump.sh |grep -i topology|awk -F ':' '/tplg/ {print $NF;}'|tail -n 1)
+tplg_file=$(journalctl -k |grep -i topology |awk -F ':' '/tplg/ {print $NF;}'|tail -n 1)
```

Comment:
Impressive simplification but I don't think it's related to the checkpointing change, is it? Can it be submitted in a later PR?